.github/workflows/tests.yml

```diff
@@ -0,0 +1,43 @@
+name: kulala tests
+on:
+  push:
+    branches:
+      - main
+  pull_request: ~
+
+  tests:
+    name: tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-22.04
+            rev: v0.10.0/nvim-linux64.tar.gz
```

Comment:
`neotest` does testing on a nightly branch. I don't think this is so interesting, so I removed the nightly branch. The way how `neotest` did this means the cache key is invalidated every day, making cache action less useful.